### PR TITLE
feat: LSDV-5520: Project Creation Wizard Tip (Label Studio Community)

### PR DESCRIPTION
**Pull request by nick-skriabin**
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: label_studio/frontend/src/components/HeidiTips/HeidiTip.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        </Elem>
+      </Elem>
+      <Elem name="heidi">
+        <HeidiSpeaking />
+      </Elem>
+    </Block>
+  );
+};
+
+const HeidiLink: FC<{ link: Tip["link"] }> = ({
```
Comment (bmartel) [inline label_studio/frontend/src/components/HeidiTips/HeidiTip.tsx]: Can we move this into source order, so it is defined before HeidiTip.

Comment (juliosgarbi) [reply label_studio/frontend/src/components/HeidiTips/HeidiTip.tsx]: done! :)

**Review by bmartel — COMMENTED** [review]
Looks good to me so far, I think this is an initial pass and there will be a follow up to fill in some of the param details correct?

**Review by bmartel — APPROVED** [review]

**Review by vladimirheartex — APPROVED** [review]

**Comment by juliosgarbi** [comment]
/git merge develop
> Successfully pushed new changes:
> Merge remote-tracking branch 'origin/develop' into fb-lsdv-5520/project-creation-tip (80f6a1c33ff16a02eae17516f5b915dec8cefbb9)
>
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/5939660547)

**Pull request merged**